Tests/AdyenTests/Adyen Tests/Core/PaymentMethodTests.swift

```diff
@@ -115,7 +116,7 @@ class PaymentMethodTests: XCTestCase {
         
         // Regular payment methods
         
-        XCTAssertEqual(paymentMethods.regular.count, 20)
+        XCTAssertEqual(paymentMethods.regular.count, 21)
```

Comment:
can we compare the count to `dictionary["paymentMethods"].count`? 
also in line 71